src/tooltip/index.tsx

```diff
@@ -3,14 +3,28 @@ import * as RMWC from '@rmwc/types';
 import { classNames, createComponent, Tag } from '@rmwc/base';
 import { useProviderContext } from '@rmwc/provider';
 import { useToolTipFoundation } from './foundation';
-import { AnchorBoundaryType, CssClasses } from '@material/tooltip';
+import { AnchorBoundaryType } from '@material/tooltip';
 
 export type TooltipActivationT = 'hover' | 'click' | 'focus';
+export type TooltipAlignT =
```

Comment:
If you move this to a shared types file, then you can import it an use it in `constants.ts`.